build.gradle.kts

```diff
@@ -13,296 +16,50 @@
  * permissions and limitations under the License.
  */
 
-import com.github.spotbugs.snom.Effort
-import org.jreleaser.model.Active
-import org.gradle.api.tasks.testing.logging.TestExceptionFormat
-
 plugins {
-    `java-library`
-    `maven-publish`
-    signing
-    checkstyle
-    jacoco
-    id("com.github.spotbugs") version "6.0.8"
-    id("io.codearte.nexus-staging") version "0.30.0"
-    id("me.champeau.jmh") version "0.7.2"
-    id("com.github.johnrengelman.shadow") version "7.1.2"
-    id("org.jreleaser") version "1.12.0" apply false
+    id("org.jreleaser") version "1.15.0" apply false
```

Comment:
could we add an alias here and move this into version library? Having everything in one place makes the dependabot changes tiny.